ocs_ci/deployment/ibm_deployment.py

```diff
@@ -0,0 +1,1021 @@
+"""
```

Comment:
please call the file `ibm` instead of `ibm_deployment.py` to keep consistency with the other files in this dir

Reply:
I will make this change along with others in this review and update pull request.

Reply:
Completed. I moved ibm_deployment.py to ibm.py